### PR TITLE
feat: update the sink metada format

### PR DESCRIPTION
**Pull request by jinfahua**

### CODE REVIEW

File: plugins/sinks/taos.json [hunk 1]
```diff
@@ -54,12 +54,12 @@
 		"control": "text",
 		"type": "string",
 		"hint": {
-			"en_US": "User of Taosdb",
-			"zh_CN": "Taosdb 用户名"
+			"en_US": "User name",
```
Comment (ysfscream) [inline plugins/sinks/taos.json:57]: `Username` should be a complete word

File: plugins/sinks/taos.json [hunk 2]
```diff
@@ -54,12 +54,12 @@
...
+			"zh_CN": "用户名"
 		},
 		"label": {
-			"en_US": "DB user",
-			"zh_CN": "数据库用户名"
+			"en_US": "User name",
```
Comment (ysfscream) [inline plugins/sinks/taos.json:61]: `Username` should be a complete word

File: plugins/sinks/taos.json (hunk 1 above)
Comment (jinfahua) [reply plugins/sinks/taos.json]: Updated.

File: plugins/sinks/taos.json (hunk 2 above)
Comment (jinfahua) [reply plugins/sinks/taos.json]: Updated.

**Review by ysfscream — APPROVED** [review]

**Pull request merged**